### ISSUE TITLE
rake eol:db:rebuild doesn't works

### ISSUE THREAD

**Issue by anthonybocci** [EOL/eol#121]
Hello,

I'm trying to install EOL on my Ubuntu 15.05, and the command "rake eol:db:rebuild" doesn't works.
The error is following :

** ERROR: Virtuoso Connection refused: undefined method `each_solution' for #<RDF::NTriples::Reader:0x00000007894020>
An Exception was thrown by scenario: bootstrap
rake aborted!

**Comment by Wala2Elsayed** [comment]
@anthonybocci : what is your virtuoso version ?

**Comment by anthonybocci** [comment]
@Wala2Elsayed my virtuoso version is 7.2.0. Should I install 7.1 version ?

**Comment by Wala2Elsayed** [comment]
@anthonybocci  We are using virtuoso version (07.10.3207). Please try it.

**Comment by anthonybocci** [comment]
@Wala2Elsayed with homebrew I can't find virtuoso (07.10.3207), should I try to install 6.16 ?

**Comment by Wala2Elsayed** [comment]
@anthonybocci  try this link https://codeload.github.com/openlink/virtuoso-opensource/zip/v7.1.0

**Comment by anthonybocci** [comment]
@Wala2Elsayed are you sure it works ? Because during command 'make' I have the following error :
Makefile:15: *** missing separator. Stop

**Comment by Wala2Elsayed** [comment]
@anthonybocci  Yes, I'm sure and i didn't get this error before.

**Comment by anthonybocci** [comment]
@Wala2Elsayed  At line 15 I've @SET_MAKE@, I think the file is an autoconf file, not a makefile. Maybe I do something of wrong during ./configure

**Comment by Wala2Elsayed** [comment]
@anthonybocci  Did you check versions of autoconf  and automake and set the required environment flags as stated in the README file?

**Comment by anthonybocci** [comment]
Yes it was because of autoconf, the version written in the Readme is wrong.
But now the problem is 'make' command works without stop, I think it's because of libtool. I'll verify

**Comment by anthonybocci** [comment]
To resolve the libtool problem :
I had to launch command `export echo=echo` because libtool use $echo.

Then, the macro AM_PATH_XML2(2.4.0) is no longer supported, instead use 
PKG_CHECK_MODULES([XML2], [libxml-2.4]) in ./configure at line 33511

**Comment by anthonybocci** [comment]
Finally I installed Virtuoso 7.1, but without your link. I cloned the [openlink repository](https://github.com/openlink/virtuoso-opensource), checkout to branch stable/7.1.0, and it compiled successfully !
But now I have the right version, and I have again an error :

ERROR: Virtuoso Connection refused: expected subject in line 2: "{ \"head\": { \"link\": [], \"vars\": [\"graph\"] },"

**Comment by Wala2Elsayed** [comment]
@anthonybocci No, I didn't get this error before.

**Comment by anthonybocci** [comment]
@Wala2Elsayed witch version of rake have you, please ?

**Comment by anthonybocci** [comment]
It doesn't works with 10.3.1

**Comment by anthonybocci** [comment]
I have rake 10.3.1 and it doesn't works.

**Comment by Wala2Elsayed** [comment]
@anthonybocci  our rake version (10.3.1, 10.4.2, 10.1.0).
